server/controllers/post.controller.js

```diff
@@ -92,5 +92,23 @@ function recommendations(req, res, next) {
   });
 }
 
+function upvote(req, res, next) {
+  const userIdString = req.user._id.toString();
+  const postIdString = post._id.toString();
+  if (req.liked) {
+    raccoon.liked(userIdString, postIdString);
+  } else if (req.unliked) {
+    raccoon.unliked(userIdString, postIdString);
+  }
+}
+
+function downvote(req, res, next) {
```

Comment:
These should be a part of the model. You shouldn't have to pass in params like this. The post should ask itself,  "am I already downvoted, if so, I am toggling inactive"

Reply:
This part of the refactor was my least favorite.  It does make sense to move these to the model.